roles/seqreports/defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+origin: seqreports
+checkqc_version: 3.6.5
+checkqc_repo: https://github.com/Molmed/checkQC
+checkqc_config_path: "{{ anaconda_path }}/envs/{{ seqreports_venv_name }}/checkqc.config"
+checkqc_bcl2fastq_output_path: Unaligned
+
+seqreports_data_path: "{{ sw_path }}/seqreports"
+
+seqreports_venv_name: seqreports-env
+seqreports_user: seqreports  # Only used in dev and local environment
+seqreports_user_uid: 506  # Only used in dev and local environment
+seqreports_group: 'MOLM-GG-BioInfo'  # Only used in dev and local environment
+seqreports_group_uid: 504
+
+seqreports_data_fastq_screen_db_path: "{{ seqreports_data_path }}/FastQ_Screen_Genomes"
+seqreports_nxf_temp: "{{ seqreports_data_path }}/nxf_tmp/"
+seqreports_nxf_work: "{{ seqreports_data_path }}/nxf_work/"
+seqreports_nxf_logs: "{{ seqreports_data_path }}/nxf_logs/"
+
+# When called from miarka, this is refering to the resource directory
+# to avoid duplicating the database for each new version.
+# Seqreports role can also be called from sysman, where this varibale is really
+# pointing to a local path
+#
+# NOTE! deployment_remote_path is not included in the sync directory!!
+seqreports_fastq_screen_resource_path: "{{ deployment_remote_path }}/resources/seqreports"
```

Comment:
unless the genome files are themselves versioned in some sensible way (which I think they are), you should probably version this path according to checkqc/seqreports or fastq_screen version (whichever is appropriate)

Reply:
We have discussed this, but we actually couldn't find any version for fastq screen.

Reply:
ok. I still think it would be good if this path could be versioned, otherwise there is always the possibility that the actual database files could change without being possible to detect. question is what to use, since the databases can be independent of software versions. I don't think I have any clear answer.. perhaps we just need to accept this and rely on the upstream versioning?

Reply:
Good point. I haven't really deep dived into it to see if it is possible to find some kind of version of the database, maybe you have @GitEdvard? This is however a risk we currently are living with in the current setup (even though it perhaps it a bit safer on our own servers?), so maybe we can make this into a separate ticket.

Reply:
:+1: works for me

Reply:
I've added a ticket for this https://snpseq.atlassian.net/browse/DEVELOP-1897